components/automate-ui/src/app/components/pending-edits-bar/pending-edits-bar.component.scss

```diff
@@ -16,9 +16,16 @@
   }
 
   button {
+    margin: 8px;
+    border: 1px solid $chef-white;
     position: absolute;
     right: 0;
     top: 0;
   }
+
+  button:hover,
+  button:focus {
+    background-color: #174AF0;
```

Comment:
More of a mental note....preaching to the choir here - this should probably become a variable at some point.  I think its fine for now though.